src/components/program/ProgramHeader.jsx

```diff
@@ -1,17 +1,58 @@
 import React, { useContext } from 'react';
-
-import { Container } from '@edx/paragon';
+import { Parallax } from 'react-parallax';
```

Comment:
This is the same package that is being used in the [prospectus](https://github.com/edx/prospectus/blob/0af9c2932e2816e49eb71c0aa1eb22b409436152/src/components/Program/ProgramHeader.jsx#L6).